### PR TITLE
Use a shell script to restart ChaosBot instead of python

### PR DESCRIPTION
**Pull request by Ameliorate**
This allows things such as redirecting stdout and stderr to a file, as well as making certain shell actions easier.

In the future, https://docs.python.org/2/library/simplehttpserver.html or the such can be used to host the stdout of the bot for easier debugging. 

Do also note that this is mostly untested, and should be reviewed before being merged.

### CODE REVIEW

**Comment by trentstollery** [comment]
👍

**Review by rtzoeller — APPROVED** [review]

**Comment by rtzoeller** [comment]
👍

**Comment by Ameliorate** [comment]
`exec > >(tee -a stdout.txt)` could be used to redirect stdout to a file by putting it on the first line of start.sh, for when we want to do that.

**Comment by amoffat** [comment]
I should elaborate that chaosbot is currently running under [supervisor](http://supervisord.org/) on its server.  This is its config file:

```
[program:chaos]
command=/root/.virtualenvs/chaos/bin/python chaos.py
directory=/root/workspace/Chaos
user=root
```

stderr and stdout are currently logged to `/var/log/supervisor/chaos-std*`.  Personally, I think it's good to leave chaos under the watchful eye of supervisor, so :-1: from me

**Comment by Ameliorate** [comment]
Thanks for the feedback, I'll use that file when I get on my desktop and can write some more advanced code.

File: chaos.py [hunk 1]
```diff
@@ -19,6 +19,11 @@
 import patch
 
 
+if not os.path.isfile("usestartscript"):
+    open("usestartupscript", 'a').close()
```
Comment (eighthjouster) [inline chaos.py:23]: Shouldn't it be `usestartscript` like in the previous line?

Comment (Ameliorate) [reply chaos.py:23]: Nice catch, but this probably isn't getting approved so I won't bother to fix that.

I'll close the PR since it looks so.

**Pull request closed without merging**